build.gradle

```diff
@@ -1180,6 +1180,16 @@ project('test:system') {
             systemProperty "imageVersion", "INVALID_IMAGE_VERSION"
         }
 
+        // validating that bookkeeper operator version >= 0.1.3
+        if (System.getProperty("bookkeeperOperatorVersion") ==~ /^(0)\.(1)\.(0|1|2)(?:-.+)/) {
```

Comment:
Can we have a more informative failure message in case any of these 2 checks fail?

Reply:
Corrected